subsys/bluetooth/host/adv.c

```diff
@@ -1437,6 +1437,33 @@ int bt_le_adv_stop(void)
 }
 
 #if defined(CONFIG_BT_PERIPHERAL)
+static inline bool adv_is_directed(struct bt_le_ext_adv *adv)
```

Comment:
Consider using this for the other places in `adv.c` where we are doing the `BT_ADDR_LE_ANY` comparison :)